From 384eeedd2d24b6953a8591524edde810ab880883 Mon Sep 17 00:00:00 2001
From: Marcos Mello <marcosfrm@gmail.com>
Date: Fri, 6 Apr 2018 08:19:27 -0300
Subject: [PATCH] fs-lib: install crc32c for ext4

EXT4 filesystems created with metadata_csum (enabled by default in mke2fs 1.44+) or ea_inode need crc32c.

https://git.kernel.org/pub/scm/fs/ext2/e2fsprogs.git/commit/?id=d0b9e0a6aa7d6805338a43b4e372623352d8df09
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git/tree/fs/ext4/super.c?h=v4.15.15#n3491
---
 modules.d/99fs-lib/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/99fs-lib/module-setup.sh b/modules.d/99fs-lib/module-setup.sh
index e614fe5f5..b69277e24 100755
--- a/modules.d/99fs-lib/module-setup.sh
+++ b/modules.d/99fs-lib/module-setup.sh
@@ -38,7 +38,7 @@ echo_fs_helper() {
 include_fs_helper_modules() {
     local dev=$1 fs=$2
     case "$fs" in
-        xfs|btrfs)
+        xfs|btrfs|ext4)
             instmods crc32c
             ;;
         f2fs)
